CHANGES.rst

```diff
@@ -1,3 +1,9 @@
+0.7.9 (unreleased)
+------------------
+
+- Added support for materialized views
+  (`AWS Docs <https://docs.aws.amazon.com/redshift/latest/dg/materialized-view-overview.html`_)
```

Comment:
This should be added under the existing 0.7.8 section, since that release has not yet been finalized and this change will go out with 0.7.8.

Reply:
okay will do.

Reply:
done.